package.json

```diff
@@ -217,6 +217,11 @@
           "type": "string",
           "default": "vX_X_X",
           "description": "Defines the protocol version to use"
+        },
+        "legend.queryBuilder.enableGridEnterpriseMode": {
+          "type": "boolean",
+          "default": false,
+          "description": "Enable AgGrid Enterprise features in the query builder (requires an AgGrid Enterprise license)"
```

Comment:
Is there a scenario where a license is provided using `legend.agGridLicense` configuration, and this is set to false?

Since I was thinking that we can infer this toggle base on the existence of `legend.agGridLicense`.

Reply:
Updated to not have a separate setting and to just set the query builder state value based on if AG_GRID_LICENSE value is present.